stacks/upload.prx.org.yml

```diff
@@ -42,6 +42,18 @@ Resources:
         Type: AWS_PROXY
         IntegrationHttpMethod: POST
         Uri: !Sub arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${UploadLambdaFunction.Arn}/invocations
+        IntegrationResponses:
```

Comment:
It's hard for me to say if API Gateway will apply the headers without the template and model definitions. I would maybe add them just for completeness.

The other thing that needs to happen to make this all work is we need to make sure the CD deploy creates a new API Gateway _deployment_. The easiest way to do that is change the name of the key on L85 and update the `DeploymentId` ref on L98 to match.

Reply:
@farski ok I added the template, model and deployment id changes. PTAL.